waffle_utils/dataset/fields/annotation.py

```diff
@@ -12,10 +12,10 @@ def __init__(
         img_id: int,
         # optional
         cat_id: int = None,
-        bbox: list[int] = None,
+        bbox: list[float] = None,
```

Comment:
Why did you change from int to float?

Reply:
I didn't know the types of coco format, so I did set the format to int with my common sense.
But, it was float type. 😂